src/App.js

```diff
@@ -14,9 +14,20 @@ export default class App extends React.Component {
     // sql.js needs to fetch its wasm file, so we cannot immediately instantiate the database
     // without any configuration, initSqlJs will fetch the wasm files directly from the same path as the js
     // see ../config-overrides.js
-    initSqlJs()
-      .then(SQL => this.setState({ db: new SQL.Database() }))
-      .catch(err => this.setState({ err }));
+
+    const me = this;
+    Promise.all([initSqlJs(), axios.get('./test.db', {responseType: 'arraybuffer'})]).then(res => {
+      const SQLite = res[0], dbStorage = res[1];
+      const db = new SQLite.Database(new Uint8Array(dbStorage.data));
+      // language=SQLite
+      // const rows = db.exec("SELECT count(*) FROM db_articles");
+      // console.log(rows);
+      me.setState({db: db});
+
+    }).catch(err => {
+      me.setState({err});
+    });
```

Comment:
```suggestion
    Promise.all([
        initSqlJs(), 
        fetch('test.db', {responseType: 'arraybuffer'})
    ]).then(([SQLite, dbStorage]) => {
      this.setState({db: new SQLite.Database(new Uint8Array(dbStorage.data))});
    }).catch(err => {
      this.setState({err});
    });
```

Reply:
@lovasoa, thank you for the advise to use `fetch`. I will use it.